src/core.rs

```diff
@@ -57,6 +57,10 @@ const CRUST_DEFAULT_BEACON_PORT: u16 = 5484;
 /// the same time.
 const MAX_JOINING_NODES: usize = 1;
 
+/// Time (in seconds) after which a joining node will get dropped from the map
+/// of joining nodes.
+const JOINING_NODE_TIMEOUT_SECS: i64 = 5;
```

Comment:
That seems a bit optimistic. I'm happy to merge this for now, but we'll need to test how long it takes nodes to join on the droplets, and then probably use again several times that limit for the actual network.